--- src/bin-win32/Makefile.am.orig	Mon Jun 12 16:48:23 2023
+++ src/bin-win32/Makefile.am	Tue Jun 20 12:28:11 2023
@@ -69,13 +69,13 @@
 if ENABLE_FFCS
 win32-dll-target: FreeFem++.exe ff-c++
 	-for i in $(LDADD2) ; do case $$i in *.dll) echo cp -p $$i .; cp -p $$i . ;; esac ; done
-	./copy-dll.sh
+	bash -c "./copy-dll.sh"
 	echo "on a fini"
 else
 win32-dll-target: FreeFem++.exe FreeFem++-nw.exe bamg.exe cvmsh2.exe launchff++.exe ff-c++ $(MPIPROG)
 	-cp ../nw/ffglut.exe . 2> /dev/null
 	-for i in $(LDADD2) ; do case $$i in *.dll) echo cp -p $$i .; cp -p $$i . ;; esac ; done
-	./copy-dll.sh
+	bash -c "./copy-dll.sh"
 	echo "on a fini"
 endif
 
